cypress/integration/delete-post.test.ts

```diff
@@ -0,0 +1,78 @@
+import { HttpMethod } from "@lib/http-method";
+import { StatusCodes } from "http-status-codes";
+import { TEST_POST_2_UUID } from "../support/constants";
+
+describe("Delete post", () => {
+  beforeEach(() => {
+    cy.task("db:setup");
+  });
+
+  afterEach(() => {
+    cy.task("db:teardown");
+  });
+
+  it("should allow a user to delete a post from the account page", () => {
+    cy.login();
+
+    cy.visit("/account");
+
+    cy.intercept(HttpMethod.DELETE, "/api/post/*").as("deletePost");
+
+    // Ensure that user posts were received
+    cy.intercept(HttpMethod.GET, "/api/user/*").as("getUserPosts");
+    cy.wait("@getUserPosts").then(({ response }) => {
+      expect(response?.statusCode).to.eq(StatusCodes.OK);
+    });
+    const posts = cy.findByRole("main").children().eq(0).children();
+
+    // Check out that posts are being generated correctly
+    posts.should("have.length", 17);
+    posts.eq(0).should("contain", "TEST POST 14");
```

Comment:
This seems brittle

Reply:
Ya, I agree. There is not a better way to confirm the post though. If I think of something, I'll change it.